bodhi/client/__init__.py

```diff
@@ -370,11 +370,25 @@ def edit(user, password, url, **kwargs):
             resp = client.query(**query_param)
             title = resp['updates'][0]['title']
         elif re.search(bindings.UPDATE_TITLE_RE, kwargs['update']):
+            query_param = {'like': kwargs['update']}
+            resp = client.query(**query_param)
             title = kwargs['update']
         del(kwargs['update'])
         kwargs['builds'] = title
         kwargs['edited'] = title
 
+        data = munch.unmunchify(resp)
+        if 'updates' in data:
+            former_update = data['updates'][0]
+        else:
+            former_update = data
+
+        # Replace None params (not provided) with previous database values
+        for k in kwargs:
+            if not kwargs[k] and k in former_update:
+                if former_update[k]:
```

Comment:
This will not work when `former_update[k] is Flase` . I don't think we need a test here.

Reply:
Yes, will modify to just include None. Although, I think we might need this test because the line above only checks for key k in the dictionary. So if there is a None value by default and we try to update that to something else, it will again be replaced by None.